src/test/java/net/minecraftforge/commontest/ForgeConfigSpecTest.java

```diff
@@ -63,6 +63,22 @@ public void deepKeySpeedTest() throws IOException
         executeSpeedTest("test.test.test.test.test.test.test.test.test.test", "deepKeyValue", "deepKeySpeedTest");
     }
 
+    @Test(expected = IllegalStateException.class)
+    public void allEmptyCommentTest() throws IllegalStateException
+    {
+        final ForgeConfigSpec.Builder builder = new ForgeConfigSpec.Builder();
+        final ForgeConfigSpec.ConfigValue<String> simpleValue = builder.comment("", "").define("allEmptyCommentTest", "someDefaultValue");
+        final ForgeConfigSpec spec = builder.build();
+    }
+
+    @Test
+    public void topPaddedCommentTest() throws IllegalStateException
```

Comment:
A bit odd to declare a runtime exception in the `throws` declaration of a method, especially in a test which does not expect this exception to be thrown.

Reply:
This is in line with the other tests in this file.

Reply:
`IOException` is a checked exception, and is therefore required to be present in the `throws` declaration of a method by [the rules of the language](https://docs.oracle.com/javase/specs/jls/se17/html/jls-8.html#jls-8.4.6). `IllegalStateException` is an unchecked exception, by virtue of extending `RuntimeException`, and therefore does not need to be declared via `throws`.